org.js.nuclear.Nuclear.json

```diff
@@ -2,9 +2,8 @@
     "app-id": "org.js.nuclear.Nuclear",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "19.08",
-    "branch": "19.08",
+    "branch": "0.6.3",
```

Comment:
this is either not set (missing), or set to `stable`.

Reply:
updated to stable although is there a pre release tag?

Reply:
This just means that it is the main branch for packages.